docs/content.zh/documentation/flink-table-store-0.3.md

```diff
@@ -1,7 +1,7 @@
 ---
-weight: 10
-title: Table Store Master (snapshot)
```

Comment:
Maybe we can just delete this file?
We just need to mention Paimon now.

Reply:
Since flink-table-store-0.3 is the last version when Paimon is part of Flink and cannot be known from Apache Paimon's website. I prefer to keep the link here if someone wants to review the history of Apache Paimon.